package.json

```diff
@@ -11,8 +11,8 @@
 		"docker-dev-vm": "rimraf node_modules/.cache/hard-source && cross-env NODE_ENV=development node server/dev-server.js",
 		"inspect-dev": "cross-env NODE_ENV=development node --inspect --max-old-space-size=4096 server/dev-server.js",
 		"inspect": "cross-env NODE_ENV=production node --inspect --max-old-space-size=4096 server/index.js",
-		"build": "rimraf dist && node build/build.js",
-		"build:cypress": "rimraf dist && node build/build.cypress.js",
+		"build": "rimraf dist && node --max-old-space-size=4096 build/build.js",
+		"build:cypress": "rimraf dist && node --max-old-space-size=4096 build/build.cypress.js",
```

Comment:
This is likely what will help. The Github actions run on a VM with 7GB of ram almost all of the failures were hitting the sweep function at 2GB.

We may also want to create a `start:cypress` command that can run at 3-4GB too...